Jenkinsfile

```diff
@@ -49,6 +49,7 @@ for (i = 0; i < buildTypes.size(); i++) {
                 '--update-snapshots',
                 "-Dmaven.repo.local=$m2repo",
                 '-Dmaven.test.failure.ignore',
+                '-DforkCount=0.5C',
```

Comment:
I do not know how many cores our container test environments have, so I am unable to assess whether this is reasonable or not. I do know for sure that setting this to 2 would retain existing behavior for the `test/` module (no regressions) and would likely not impact the `core/` module, which only takes a few minutes to run anyway, (no regressions). So while I would be a +1 for a value of 2, I am undecided on a value of 0.5C pending an explanation of what that would translate to in practice on our container agents.

Reply:
In practice they seem equivalent (2 tests in parallel). https://github.com/jenkins-infra/documentation/blob/main/ci.adoc#jenkins-on-jenkins doesn't say anything about specs of default containers (unlike VMs), would probably be valuable.

Reply:
But it could easily change in the future and cause unexpected breakage. I think an explicit value of 2 would be less fragile.

Reply:
> [https://github.com/jenkins-infra/documentation/blob/main/ci.adoc#jenkins-on-jenkins](https://github.com/jenkins-infra/documentation/blob/main/ci.adoc?rgh-link-date=2022-09-15T18%3A57%3A00Z#jenkins-on-jenkins) doesn't say anything about specs of default containers (unlike VMs), would probably be valuable.

This page needs an update indeed.

[Here and below](https://github.com/jenkins-infra/jenkins-infra/blob/9804c214dd700237b3a830225019908765ae9952/hieradata/clients/azure.ci.jenkins.io.yaml#L172-L218) you can see the number of CPU and mem allocated to the kubernetes agents.